ding_unilogin/ding_unilogin.module

```diff
@@ -240,6 +248,10 @@ function ding_unilogin_publizon_client_id_alter(&$client_id) {
 
 /**
  * Whether to globally log out.
+ *
+ * @param null $state
```

Comment:
I suppose that `$state` can be something other that `null`.

Reply:
Fixed.